cetmix_tower_server/models/cx_tower_server.py

```
@@ -1148,12 +1323,17 @@ def _command_runner(
                 rendered_command_code,
                 **kwargs,
             )
+        elif command.action == "jet_action":
+            response = self._command_runner_jet_action(
+                log_record,
+                **kwargs,
+            )
```

Pylint (4.0.4)

[error] Assigning result of a function call, where the function returns None

(E1128)